common/djangoapps/student/management/commands/change_enrollment.py

```diff
@@ -40,11 +52,16 @@ class Command(BaseCommand):
                     dest='to_mode',
                     default=False,
                     help='move to this enrollment mode'),
-        make_option('-u', '--user',
-                    metavar='USER',
-                    dest='user',
+        make_option('-u', '--username',
```

Comment:
Might make this `--usernames` since it now supports multiple users. Same with `emails` below.